libcylinder/src/key/load.rs

```diff
@@ -246,7 +246,7 @@ mod tests {
     /// 3. Ensure the private key is returned.
     #[test]
     fn load_key_success() {
-        let temp_dir = TempDir::new("test_key_dir").expect("Failed to create temp dir");
+        let temp_dir = TempDir::new().expect("Failed to create temp dir");
```

Comment:
In order to retain the prefix, this should be Builder::new()::prefix("test_key_dir").tempdir().expect("Failed to create temp dir"); (with a use tempfile::Builder above).